sections/plate_notation.tex

```diff
@@ -0,0 +1,39 @@
+\section{Plate Notation}
+As with \citet{author_topic_2012}, there are no document-topic distributions $\theta$ in our metadata models.
+Instead, each author and category has their own topic distribution.
+This is based on the assumption that authors prefer to write about specific topics, and that categories of the articles were chosen based on the content of the finished article or that local newspapers have their own unique topic preferences.
+
+Our topic models are \glspl{um}, meaning that topics are only word distributions and are chosen based on the data of the documents, rather than \glspl{dm} where topics are fitted to have influence on both word distributions and other metadata.
+
+Our standard \gls{lda} model uses a similar notation to \citet{blei2003latent}, where a document $d$ is a vector of $N$ ID's, where the ID represents the given word in the original document. $Z_{d,n}$ is the topic assigned to a word $n$ within $d$.\todo[inline]{In the original LDA paper, a document is actually a sequence of N words, where each word is a one-hot vector encoding. Might not be important to change, since we just say the notation is 'similar'.}
+$\theta_d$ denotes the document-topic distribution for a specific document $d$ and the $\beta_k$ denotes the topic-word distribution for a specific topic $k$.
+The plate notation for \gls{lda} can be seen in \autoref{fig:standard_lda}.
+\todo[inline]{Some of this text about our use of standard LDA can maybe be cut down, if we have a specific LDA section and simply ref to that.}
+
+\citet{author_topic_2012} describes the author-topic model where for each document $d$, they assign a vector of authors $a_d$ from a set of authors $A$, and for each word draw an author $x$ from this vector.
+
+In our category-topic model and our own author-topic model, we use a similar approach to the author-topic model from \citeauthor{author_topic_2012}, where each document $d$ is associated with a category $c_d$ from a set of categories $C$ or an author $a_d$ from a set of authors $A$.
+The plate notation for our category and author \gls{lda} models can be seen in \autoref{fig:metadata_lda}.
```

Comment:
we should also mention that we only account for one author / category per document, rather than a vector, as with the original author-topic model